python/cudf/cudf/core/dataframe.py

```diff
@@ -3333,7 +3333,11 @@ def agg(self, aggs, axis=None):
 
     @_cudf_nvtx_annotate
     def nlargest(self, n, columns, keep="first"):
-        """Get the rows of the DataFrame sorted by the n largest value of *columns*
+        """Return the first *n* rows ordered by *columns* in descending order.
+
+        Return the first *n* rows with the largest values in *columns*, in
+        descending order. The columns that are not specified are returned as
+        well, but not used for ordering.
```

Comment:
This is a copy of the equivalent pandas docstring, but it is rather awkward.

```
Return the largest *n* rows ordered by *columns* in descending order.

All columns of the dataframe are returned, but only those specified in *columns*
are used to construct the key used for ordering.
```

WDYT?

Reply:
I prefer exactly matching pandas. It is the easiest way for developers and users to be assured that the behavior is meant to be identical between the two libraries.